### PR TITLE
Discussion to update web defaults

### PR DESCRIPTION
**Pull request by hhsnopek**
This has been the default I've been using for a while as on many projects.We haven't utilized and/or have a better flow due to `zenhub`.
```
feature-request:adc6c8
in-progress:f4f7eb
needs-review:d9e2b8
on-hold:939b75
```

`feature-request`, is a feature that can be open/closed based on discussion in the thread. I've never seen this used. `in-progress`, is a label that we have a zenhub pipeline for. `needs-review`, we have the `Reviewers` option on pull requests as well as a pipeline for QA. `on-hold`, typically ends up being `blocked`. 

I'm no longer importing the `carrots` base file as I don't want to interrupt other workflows in `ios`/`android` that may be utilizing these labels.

### CODE REVIEW

**Comment by BrandonRomano** [comment]
I use the `feature-request` tag to keep track of all non-scope issues, so at the end of the project I can look to see "Hey, how many things have we already done for them beyond scope".. Helps our account people argue against scope creep.  I argue to keep this.

These three things:

```
in-progress:f4f7eb
needs-review:d9e2b8
on-hold:939b75
```

I agree we should remove `in-progress` and `needs-review`, but we should remove it from the `carrots` base one not from `carrots-web`.

And `on-hold` I still vote to remove it.  A TPM could want to temporarily put something on hold without fully blocking it.... Usually I use `on-hold` to be like "Don't do this until we have a conversation" but it's still not blocked.

TLDR: I vote to remove these from the carrots base:

```
in-progress:f4f7eb
needs-review:d9e2b8
```

**Comment by hhsnopek** [comment]
hmm do other workflows currently use `in-progress/needs-review`?
in regards to `on-hold` I see your point, personally haven't used it, but lets keep it 😄

File: extras/label_configs/carrots-web [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Carrots Base
+# reset
```
Comment (BrandonRomano) [inline extras/label_configs/carrots-web]: This comment is incorrect now!

File: extras/label_configs/carrots [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Remove Github defaults
--import:clear-defaults
+-delete:all
```
Comment (BrandonRomano) [inline extras/label_configs/carrots]: This is a little more dangerous than clear-defaults..

Do we need to change this?

Comment (hhsnopek) [reply extras/label_configs/carrots]: So unless we do this, there could be other labels that were setup previously that wouldn't be removed. It clears the edge case of something initializing with `carrots-ios` and then switching over to another label set.

Comment (hhsnopek) [reply extras/label_configs/carrots]: Maybe we need a better way to `fix` labels on Github in the event of a change?
Not sure how this would interface considering we don't track what the repo label set is, might be able to do:

```
ash github:labels [fix/change] [labelConfig] [org/repo]
  > Are you sure you want to change the labels in [org/repo] (Y/n): 
```

Comment (BrandonRomano) [reply extras/label_configs/carrots]: So, I think it is an edge case, as you're really supposed to use this lib right when a repo starts off.

If you're calling this in any other time than that, we shouldn't be deleting labels as we could lose important tags.

Comment (hhsnopek) [reply extras/label_configs/carrots]: my thought is updating repos with the wrong tags - that was the reason of adding `-delete:all` originally

Comment (hhsnopek) [reply extras/label_configs/carrots]: Also considering the usage should be at the start of the repo's history one shouldn't lose the important tags?

Comment (hhsnopek) [reply extras/label_configs/carrots]: I'm gonna change it back and move this discussion to an issue thread

**Review by BrandonRomano — APPROVED** [review]
Just squash this

**Comment by hhsnopek** [comment]
squashed
